### ISSUE TITLE
Input field: Extra leading space causes "Invalid package query" error

### ISSUE THREAD

**Issue by aerda** [rschristian/deptree#6]
## Description
When pasting text into the input field, there's a risk of accidentally including an extra space at the beginning. This leads to an "Invalid package query" error.

## Steps to reproduce:
1. Copy some text.
2. Paste the text into the input field, ensuring there's a leading space.
3. Submit the form.
4. Observe the "Invalid package query" error.

## Expected behavior:

The input field should either:

- Automatically trim leading and trailing whitespace.
- Provide clear feedback to the user about the invalid input, suggesting how to correct it.

<img width="920" alt="image" src="https://github.com/user-attachments/assets/8db0fd97-c0f0-4a61-96fe-a94603cfe229">

**Comment by rschristian** [comment]
Sure, trim sounds reasonable.

Feel free to PR or I can take a look at it within the next few days, schedule permitting.

**Comment by aerda** [comment]
setPkgQuery(
   /** @type {HTMLInputElement} */ (e.target).value.trim(),
)


```
<input
    autocorrect="off"
    autocapitalize="none"
    enterkeyhint="search"
    class="py-2 px-4 w-full text(xl md:3xl center [#111]) bg-input(& dark:dark) drop-shadow-lg rounded-lg hocus:(outline(2 & primary))"
    placeholder="Provide a package name"
    value={pkgQuery}
    onInput={(e) =>
        setPkgQuery(/** @type {HTMLInputElement} */ (e.target).value)
    }
/>
```

**Comment by aerda** [comment]
> Sure, trim sounds reasonable.
> 
> Feel free to PR or I can take a look at it within the next few days, schedule permitting.

remote: Permission to rschristian/deptree.git denied to aerda.
fatal: unable to access 'https://github.com/rschristian/deptree.git/': The requested URL returned error: 403

**Comment by rschristian** [comment]
You need to fork the repo and make the changes there, you can't push directly to projects you do not own/haven't been invited to.

Here's some instructions if you're new to GitHub:
  - [Fork a repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)
  - [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)

**Comment by rschristian** [comment]
Fixed in 4631b961bb827afba702bbee7a17b171f95b7415
